sled-agent/src/services.rs

```diff
@@ -132,10 +132,10 @@ const IPV6_UNSPECIFIED: IpAddr = IpAddr::V6(Ipv6Addr::UNSPECIFIED);
 const COCKROACH: &str = "/opt/oxide/cockroachdb/bin/cockroach";
 
 const CLICKHOUSE_SERVER_BINARY: &str =
-    "/opt/oxide//opt/oxide/clickhouse_server/clickhouse";
```

Comment:
ugh, I did this didn't I? sorry!!!

Reply:
Ha no big deal. I had assumed I did it along with the 5 other breaking changes I introduced when splitting things up :)